inference-engine/src/gna_plugin/gna_graph_compiler.hpp

```diff
@@ -36,7 +36,8 @@ class GNAGraphCompiler {
     std::shared_ptr<GNAPluginNS::InputDesc> inputDesc;
     std::shared_ptr<GNAPluginNS::GNAFlags> gnaFlags;
     Policy policy;
-
+    //consider moving to configuration
+    const bool cnn2dInputPaddingSupported = true;
```

Comment:
It should be in policy, see gna_plugin_policy.hpp

Reply:
Moved